core/src/test/java/io/confluent/kafka/schemaregistry/rest/RestApiTest.java

```diff
@@ -182,6 +183,56 @@ public void testRegisterBadDefault() throws Exception {
     }
   }
 
+  @Test
+  public void testRegisterInvalidSchemaBadType() throws Exception {
+    String subject = "testSubject";
+
+    //Invalid Field Type 'str'
+    String badSchemaString = "{\"type\":\"record\","
+            + "\"name\":\"myrecord\","
+            + "\"fields\":"
+            + "[{\"type\":\"str\",\"name\":\"field1\"}]}";
+
+    String expectedErrorMessage = "Expected Error String";
```

Comment:
Initialised it with random value so that the "contains" check doesn't pass trivially in case of no exception. I guess we can add a non-empty error message assertion instead.

Will make the change.

Reply:
Addressed.